demos/html5/ckeditor5/README.md

```diff
@@ -0,0 +1,19 @@
+# Demo for mathtype on html5
+
+Here are some instructions to work with this demo and with lerna, as it is complicated to do the same way the other demos do. That is because CKEditor 5 does not let to have duplicate modules, and this has interferences whit the demo in the development mode. 
+
+## Production mode
+
+Use one of the following commands to run the demo in the production mode:
+```sh
+$ npm start
+$ npm deploy
+$ npm build-dev
```

Comment:
All three are interchangeable? In that case, I would just write one instead of all three.